plan/config.go

```diff
@@ -104,10 +113,29 @@ func parseBehavior(d string) Behavior {
 		ClientAxis: clientAxis,
 		ParamsAxes: values,
 	}
-
 	return behavior
 }
 
+func parseSkipBehavior(d string) (string, []Filter) {
```

Comment:
Please add a comment describing the grammar and semantics for the skip string here or link elsewhere.